credentials/src/main/kotlin/web5/sdk/credentials/PresentationExchange.kt

```diff
@@ -65,6 +69,52 @@ public object PresentationExchange {
     }
   }
 
+  /**
+   * Creates a Presentation Submission against a list of Verifiable Credentials (VCs) against a specified
+   * Presentation Definition.
+   *
+   *
+   * @param vcJwts Iterable of VCs in JWT format to validate.
+   * @param presentationDefinition The Presentation Definition V2 object against which VCs are validated.
+   * @return A PresentationSubmission object.
+   * @throws UnsupportedOperationException if the presentation definition contains submission requirements.
+   * @throws IllegalStateException if no VC corresponds to an input descriptor or if a VC's index is not found.
+   * @throws PresentationExchangeError If the number of input descriptors matched is less than required.
+   */
+  public fun createPresentationFromCredentials(
+    vcJwts: Iterable<String>,
+    presentationDefinition: PresentationDefinitionV2
+  ): PresentationSubmission {
+
+    satisfiesPresentationDefinition(vcJwts, presentationDefinition)
+
+    val inputDescriptorToVcMap = mapInputDescriptorsToVCs(vcJwts, presentationDefinition)
+    val vcJwtToIndexMap = vcJwts.withIndex().associate { (index, vcJwt) -> vcJwt to index }
+
+    val descriptorMapList = mutableListOf<DescriptorMap>()
+    for ((inputDescriptor, vcList) in inputDescriptorToVcMap) {
+      // Even if multiple VCs satisfy the input descriptor we use the first
+      val vcJwt = vcList.firstOrNull()
+      checkNotNull(vcJwt) { "Illegal state: no vc corresponds to input descriptor" }
+
+      val vcIndex = vcJwtToIndexMap[vcJwt]
+      checkNotNull(vcIndex) { "Illegal state: vcJwt index not found" }
+
+      descriptorMapList.add(
+        DescriptorMap(
+          id = inputDescriptor.id,
+          path = "$.verifiableCredential[$vcIndex]",
+          format = "jwt_vc"
```

Comment:
Would it make sense to use `Format` as the type here?

Reply:
since the input to the function is vcJwts: Iterable<String>, we know that he descriptor map will be jwt_vc mapping